packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1161,8 +1161,43 @@ The Multifactor (MFA) Enrollment Policy controls which MFA methods are available
    }
 ```
 
+#### Policy Authenticators Settings example
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+> **Note:** Policy Settings are included only for those Authenticators that are enabled.
+
+```json
+   "settings": {
+     "type": "AUTHENTICATORS",
+     "authenticators": [
+       {
+         "key": "security_question",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       },
+       {
+         "key": "okta_phone",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       }
+     ]
+   }
+```
+
 ### Policy Settings data
 
+> **Note:** Default type is `FACTORS` when not specified.
+
+| Parameter                                                               | Description                                                                  | Data Type                      | Required |
+| ---                                                                     | ---                                                                          | ---                            | ---      |
+| type <ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" />  | Type of policy configuration object                                                | `FACTORS`, or `AUTHENTICATORS` | No       |
+| factors                                                                 | [Policy Factors Configuration object](#policy-factors-configuration-object)  | Array                          | No       |
+| authenticators                                                          | Array of [Policy MFA Authenticator object](#policy-mfa-authenticator-object) | Array                          | No       |
```

Comment:
Can we add the `<ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" />` tags here too